app/presenters/firm_admin/defence_request_presenter.rb

```diff
@@ -0,0 +1,48 @@
+class FirmAdmin::DefenceRequestPresenter < SimpleDelegator
```

Comment:
This is at the moment just C&P of the `DefenceRequestPresenter`. If we don't need it different from the other one, than I'd delete it and even for firm admin use the original one.